homeassistant/components/epic_games_store/const.py

```diff
@@ -0,0 +1,24 @@
+"""Constants for the Epic Games Store integration."""
+
+DOMAIN = "epic_games_store"
+
+CONF_LOCALE = "locale"
+
+CONF_SUPPORTED_LOCALES = (
+    "ar",
+    "de",
+    "en-US",
+    "es-ES",
+    "es-MX",
+    "fr",
+    "it",
+    "ja",
+    "ko",
+    "pl",
+    "pt-BR",
+    "ru",
+    "th",
+    "tr",
+    "zh-CN",
+    "zh-Hant",
+)
```

Comment:
Should this not be in the library?

Reply:
This may, but don't know if it will be accepted as it won't be used anywhere else in the API code.

As it's used here, as `alexa`, or `baidu`, `google_cloud`, `microsoft` (but with `SUPPORTED_LANGUAGES`), I think here is also nice.